tests/python_api/audio_unit_test.py

```diff
@@ -77,14 +77,15 @@ def verify_output(audio_tensor, rocal_data_path, roi_tensor, test_results, case_
     roi_data = roi_tensor.detach().numpy()
     buffer_size = roi_data[0] * roi_data[1]
     matched_indices = 0
+    atol = 1e-20 if case_name != "normalize" else 1e-5
```

Comment:
please explain why this needs to be changed for normalize

Reply:
Added comment for reduced tolerance